h2o-algos/src/main/java/hex/glrm/GLRMModel.java

```diff
@@ -392,6 +392,7 @@ public ModelMetricsGLRM scoreMetricsOnly(Frame frame) {
 
     GLRMScore gs = new GLRMScore(ncols, _parms._k, false).doAll(fullFrm);
     ModelMetrics mm = gs._mb.makeModelMetrics(GLRMModel.this, adaptedFr, null, null);   // save error metrics based on imputed data
+    mm._frame_checksum = frame.checksum();      // get the checksum of the original frame and not the one changed
```

Comment:
We calculate the checksum in makeModelMetrics for the adaptedFr and overwrite the checksum here. Perhaps we can improve it by passing the original frame to the makeModelMetrics functions and avoid the need to expose the _frame_checksum as public and overwriting it.